packages/cli-doctor/src/tools/healthchecks/xcodeEnv.ts

```diff
@@ -0,0 +1,21 @@
+import {HealthCheckInterface} from '../../types';
+import fs from 'fs';
+import {promisify} from 'util';
+
+export default {
+  label: '.xcode.env',
+  description: 'File to customize Xcode environment',
+  getDiagnostics: async () => {
+    return {
+      needsToBeFixed: fs.existsSync('./ios/.xcode.env'),
```

Comment:
This is making assumptions on where the iOS project lives. Should we use the config to figure out where the closes `Podfile` is? And what about macOS?

Reply:
I updated it using `findProjectRoot`. I'm not sure that `findPodfilePath` is the right approach, because for macOS project we have an `ios` folder and a `macos` folder, both of which have their own `Podfile`.

```sh
$ ls -all ios
total 48
drwxr-xr-x  10 cipolleschi  staff    320  6 Apr 17:00 .
drwxr-xr-x  24 cipolleschi  staff    768  6 Apr 17:03 ..
-rw-r--r--   1 cipolleschi  staff   1106  6 Apr 16:59 Podfile
// ... other files here ...

$ ls -all macos
total 56
drwxr-xr-x  10 cipolleschi  staff    320  6 Apr 17:06 .
drwxr-xr-x  24 cipolleschi  staff    768  6 Apr 17:03 ..
-rw-r--r--   1 cipolleschi  staff   1086  6 Apr 17:03 Podfile
```